tests/integration/modules/test_vsphere.py

```diff
@@ -52,3 +53,18 @@ def test_list_ssds(service_instance):
     for host_name in ret:
         for ssd in ret[host_name]:
             assert isinstance(ssd, str)
+
+
+def test_list_non_ssds(service_instance):
+    pattern = r"^[a-zA-Z0-9]{3}\."
+    ret = vsphere.list_non_ssds(
+        service_instance=service_instance,
+    )
+    assert ret
+    for host_name in ret:
+        for ssd in ret[host_name]:
+            assert isinstance(ssd, str)
+            if re.match(pattern, ssd):
```

Comment:
That assumes that all names in that regex for non ssds, which at some point will probably become untrue.